src/decision_model.jl

```diff
@@ -229,12 +230,24 @@ function expected_value(model::Model,
     diagram::InfluenceDiagram,
     x_s::PathCompatibilityVariables)
 
+    diff_sign_utilities = false
```

Comment:
Tests should be added for these new errors

Reply:
Added tests for DP expected_value and conditional_value_at_risk functions